bubblesub/model.py

```diff
@@ -229,6 +236,13 @@ def __iter__(self) -> T.Iterator[TItem]:
         """
         yield from self._items
 
+    def __reversed__(self) -> T.Iterator[TItem]:
+        """Iterate reversly over the collection values.
+
+        :return: iterator
+        """
+        yield from self._items[::-1]
```

Comment:
Me gusta, I'd do `reversed(self._items)` but I keep forgetting about those advanced slicing features

Reply:
Thank you :)

Reply:
Thanks! :)